first_assignment.py

```diff
@@ -1,22 +1,19 @@
 from PyQt5.QtWidgets import (
     QApplication, QMainWindow, QLabel, QPushButton, 
     QCheckBox, QSlider, QComboBox, QWidget, 
-    QHBoxLayout, QVBoxLayout, QLineEdit
+    QHBoxLayout, QVBoxLayout, QLineEdit, QMessageBox
 )
 from PyQt5.QtCore import Qt
+import os
 
 class MainWindow(QMainWindow):
-    """Custom window meant to display a textbox, dropdown bar,
+    """Custom window meant to display a self.textbox, self.dropdown bar,
```

Comment:
nit: you can just say textbox and dropdown bar for the comment best to leave code out

Reply:
Woops I think I accidentally typed those self's because I selected all instances of those words